ako-gateway-api/nodes/dequeue_ingestion.go

```diff
@@ -215,16 +215,20 @@ func saveAviModel(modelName string, aviGraph *nodes.AviObjectGraph, key string)
 func (o *AviObjectGraph) ProcessRouteDeletion(key, parentNsName string, routeModel RouteModel, fullsync bool) {
 
 	parentNode := o.GetAviEvhVS()
+	routeTypeNsName := routeModel.GetType() + "/" + routeModel.GetNamespace() + "/" + routeModel.GetName()
 
-	found, childVSNames := akogatewayapiobjects.GatewayApiLister().GetRouteToChildVS(routeModel.GetType() + "/" + routeModel.GetNamespace() + "/" + routeModel.GetName())
+	found, childVSNames := akogatewayapiobjects.GatewayApiLister().GetRouteToChildVS(routeTypeNsName)
 	if found {
 		utils.AviLog.Infof("key: %s, msg: child VSes retrieved for deletion %v", key, childVSNames)
 
 		for _, childVSName := range childVSNames {
 			o.RemovePoolNameFromStringGroups(childVSName, parentNode, key)
 			nodes.RemoveEvhInModel(childVSName, parentNode, key)
```

Comment:
We can remove this line as it is being called at #227.

Reply:
Done